src/psycopt2d/feature_transformers.py

```diff
@@ -0,0 +1,83 @@
+"""Contains custom transformers for data preprocessing."""
+from datetime import datetime
+from typing import List, Optional
+
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class ConvertToBoolean(BaseEstimator, TransformerMixin):
+    def __init__(
+        self,
+        columns_to_skip: Optional[List[str]] = ["age_in_years", "sex_female"],
+        columns_to_include: Optional[List[str]] = None,
```

Comment:
```suggestion
        columns_to_include: Optional[List[str]] = None,
        columns_to_skip: Optional[List[str]] = ["age_in_years", "sex_female"],
```
Changed the order since `columns_to_skip` is ignored if `columns_to_include` is set.